Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift

```diff
@@ -9,7 +9,6 @@ import Foundation
 
 public enum Sections: CaseIterable {
     case menuSandwiches
-    case ingredients
 }
```

Comment:
Are you considering adding more sections in the future? If not, you could also remove all the Section code.

Reply:
Yes I agree, I'll do it now.